library/std/src/env.rs

```diff
@@ -472,8 +461,8 @@ pub struct JoinPathsError {
 /// # }
 /// ```
 ///
-/// Using `env::join_paths` with [`env::split_paths`] to append an item to the `PATH` environment
-/// variable:
+/// Using `env::join_paths` with [`split_paths`] to append an item to the
```

Comment:
We should be consistent: either use `env::` for both or for neither.
```suggestion
/// Using `join_paths()` with [`split_paths()`] to append an item to the
```

Reply:
Nah, we need to just add more links. I didn't notice this because `join_paths` does not have square brackets.
```suggestion
/// Using [`join_paths()`] with [`split_paths()`] to append an item to the
```